### ISSUE TITLE
rMPP Support

### ISSUE THREAD

**Issue by GeorgeHahn** [rM-self-serve/webinterface-wifi#7]
I may be able to help with this if you're open to it; I have things pretty well working on my rMPP. The only unknown is I haven't tried enabling the web interface on boot yet.

My changes are [here](https://github.com/GeorgeHahn/webinterface-wifi).

Summary of changes:
- Hacked in a new SSID reading method
- Build for aarch64

**Comment by rM-self-serve** [comment]
Awesome! Was hoping to get some help with support. I should have time later this week to integrate the changes, feel free to make a pull request in the meantime. The build step won't be necessary as that is done via github actions with the toolchain container from toltec.

**Comment by GeorgeHahn** [comment]
The other devices seem to be 32 bit arm. I haven't checked if toltec has rMPP support incoming, but I believe it's going to need a different build.

**Comment by rM-self-serve** [comment]
There are containers containing the toolchain at https://github.com/toltec-dev/toolchain, but I must misremember seeing rMpp support. Both architectures will need to be built and then sorted with the installation script.

**Comment by rM-self-serve** [comment]
Could you make a pull request on the dev_rMpp branch? https://github.com/rM-self-serve/webinterface-wifi/tree/dev_rMpp

Don't worry about modifying your repo at all, I will accept the PR then continue working on the branch until it is merged to master. I just want to make sure your contribution is noted. Thanks.

**Comment by GeorgeHahn** [comment]
Hi,

Sorry I haven’t been able to do this yet. Please feel free to use this code, no credit necessary.

Cheers,
George

**Comment by rM-self-serve** [comment]
@mellisio I saw you applied these changes, can you confirm that they work? Thanks. Should be able to support rMpp this week.

**Comment by mellisio** [comment]
Yes they work. I tried to make a pull on the branch you mentioned above but I don't have permissions.

**Comment by rM-self-serve** [comment]
Would love if someone could test the new rMpp support, instructions are in the readme. Thanks :)
